app/forms/changelog_entry_form.rb

```diff
@@ -30,11 +36,16 @@ def self.from_entry(entry)
     :details_markdown,
     :referenceable_gid,
     :info_url,
-    :created_by
+    :created_by,
   )
 
+  attr_reader :created_by
+
   def save
     raise UnauthorizedUserError unless created_by.may_edit_changelog?
```

Comment:
This is problematic, because if a user has their changelog permissions removed, then no admin can ever change the changelog. I would think that `created_at` here should be the actual user changing it, which should probably be passed in as a param if this is where the security is happening now.